binaries/speed_benchmark_torch.cc

```diff
@@ -136,9 +140,15 @@ std::vector<c10::IValue> create_inputs() {
           "Unsupported input memory format: ", input_memory_format_list[i]);
     }
 
-    inputs.push_back(torch::ones(
+    auto input_tensor = torch::ones(
         input_dims,
-        at::TensorOptions(input_type).memory_format(input_memory_format)));
+        at::TensorOptions(input_type).memory_format(input_memory_format));
+    if (FLAGS_vulkan) {
+      auto input_tensor_vulkan = input_tensor.vulkan();
+      inputs.push_back(input_tensor_vulkan);
```

Comment:
You could consider using ```inputs.push_back(input_tensor.vulkan())``` instead to take advantage of .push_back()'s overload of r-value references.

Both of these comments are very minor, but in case you are interested in some tips which you are more than welcome to ignore.